### DISCUSSION TITLE
Welcome to flamerobin Discussions!

### DISCUSSION THREAD

**Discussion by mariuz** [mariuz/flamerobin#143] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wudai8** [comment]
我在使用FlameRobin工具连接Firebird数据时，出现错误，你能帮下我吗？
![84b757df2a538cbeae08b2bdd6c1a2e](https://user-images.githubusercontent.com/105644366/181592299-f5144083-68af-4816-b22b-a95f509de547.jpg)
